minetest/minetest
# [NoSquash] Replace settings tab with button

Part of #13476

![image](https://github.com/minetest/minetest/assets/2122943/c4df7468-5088-4979-9596-6715ef273b2c)
![image](https://github.com/minetest/minetest/assets/2122943/500e2d1a-118b-48ff-9e92-1c8358fb3a7c)
![image](https://github.com/minetest/minetest/assets/2122943/54c86f5e-f7d0-4d0f-94a1-2d74a121056a)



## To do

This PR is Ready for Review


## How to test

* Open main menu
* Click settings button
* See settings
